docs/use-case-descriptions/message-validation.md

```diff
@@ -13,25 +13,73 @@ If it fails validation, a response message is sent with ‘rejected’ in the re
 Some causes to reject a message are specific for the type of message and are mentioned in the remainder of this chapter.
 Other causes are more generic and are listed in the table below.
 
-This list is not all-encompassing: stakeholders are allowed to formulate other reasonable causes to reject a message, as long as it is clearly explained in the RejectionReason field.
+This list is not all-encompassing: stakeholders are allowed to formulate other reasonable causes to reject a message, as long as it is clearly explained in the RejectionReason field. Adding the cause of rejection helps the receiving implementation to quickly resolve the issue. 
```

Comment:
I would relax the claim that adding the rejection reason will quickly resolve the issue; I would say something more neutral like "Adding the cause of the rejection will facilitate in building a correct implementation"

Reply:
I modified this line to your suggestion.